util/src/main/i18n/templates/map.properties

```diff
@@ -58,6 +58,8 @@ map.setNext.revert = {0} has removed {1} as the next map
 
 map.noNextMap = No next map
 
+map.noMapsFound = No maps found
```

Comment:
If I understand this correctly, this is displayed if you do `/vote clear` and there is no maps to clear.
I think this is too vague if I am correct. Maybe something like `There are no custom vote options to clear` | `There are no maps to clear from the vote` | `No vote options to clear` | `No maps to clear`